docs/search/full-text/index.mdx

```diff
@@ -168,6 +169,26 @@ Each `_fields` input to `create_bm25()` accepts a [JSON5](https://json5.org)-for
     </ParamField>
   </Expandable>
 </ParamField>
+<ParamField body="datetime_fields">
+  A JSON5 string which specifies which datetime columns should be indexed and how they should be indexed.
+  Keys are the names of columns, and values are config options. Accepts columns of type `date`, `timestamp`,
+  `timestamptz`, `time`, and `timetz`.
+  Search terms will always be read in UTC timezone regardless of the underlying column type. Convert search
+  terms accordingly.
```

Comment:
This might warrant an example, I'm not sure users will be able to interpret these instructs easily on their own.

It sounds like even if my column is PST... the search query should be UTC too. Are the results UTC? So I need to map both query input and the SELECT output?

That's fine if so, it's understandable that we're subject to Tantivy's limitations around timezone handling, but all the more reason I think an example will be helpful.

Reply:
I added some documentation to clear this up. You can actually make a query in any time zone, but if it's not specified, it's assumed to be UTC.